frontend/src/features/mapSideBar/containers/MapSideBarContainer.tsx

```diff
@@ -259,6 +305,27 @@ const MapSideBarContainer: React.FunctionComponent<IMapSideBarContainerProps> =
       hidePolicy={true}
     >
       {render()}
+      <GenericModal
+        message={
+          <>
+            <FloatCheck size={32}></FloatCheck>
```

Comment:
This is handy, I think there are some other places we can do something similar. Also, I've always just assumed but never knew 100%. Is `React.Fragment` the same thing as doing `<>`?

Reply:
yep, its just newer syntax